flink-table/flink-table-runtime/src/test/java/org/apache/flink/table/runtime/functions/table/fullcache/inputformat/FullCacheTestInputFormat.java

```diff
@@ -41,18 +41,22 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-/** TestInputFormat that reads data from (2 + delta) splits which share the same {@code queue}. */
+/**
+ * TestInputFormat that reads data from (initNum + delta) splits which share the same {@code queue}.
+ */
 public class FullCacheTestInputFormat
         extends RichInputFormat<RowData, FullCacheTestInputFormat.QueueInputSplit> {
 
     public static final AtomicInteger OPEN_CLOSED_COUNTER = new AtomicInteger(0);
-    private static final int DEFAULT_NUM_SPLITS = 2;
+    public static final int DEFAULT_NUM_SPLITS = 2;
+    public static final int DEFAULT_DELTA_NUM_SPLITS = 0;
```

Comment:
```suggestion
    static final int DEFAULT_NUM_SPLITS = 2;
    static final int DEFAULT_DELTA_NUM_SPLITS = 0;
```

Reply:
tbh, it appears that we could even make these values `private`. The only time, they are used is in `InputFormatCacheLoaderTest` which doesn't seem to rely on these values but just uses them as "some" input value. Or is there a specific reason why we need the default values as input values? :thinking:

Reply:
Agree, changed to `private`